### ISSUE TITLE
Aria and accessibility

### ISSUE THREAD

**Issue by fontanil** [templaza/astroid-framework#573]
Hi, I tested the accessibilty of a website with Pagespeed and got this error : "Assistive technologies such as screen readers cannot interpret ARIA attributes if their values are invalid"
![image](https://github.com/templaza/astroid-framework/assets/8665970/38fa6fa1-8dcc-4c85-85b4-91720cc70288)
How to solve that?

Thanks for your help!

**Comment by sonvnn** [comment]
Which part do you get this error? Widgets, Modules or Layout? Please detail the issue to help us save time to find solution.

Thank you!

**Comment by fontanil** [comment]
Hi,

It's in a module: https://www.kennedy-formations.com/ module "les entreprises"
The site is using Astroid One

**Comment by sonvnn** [comment]
If you are using a module please help me check that module's content. Some custom html will raise issue when you copy and paste content directly without clear text.

**Comment by fontanil** [comment]
Thanks Sonny,

It's hardcoded and I don't know why. 
The text was added by my client:
`<div class="dnXCYb" tabindex="0" role="button" aria-controls="_6nPvZI36Dp-hkdUPjvqb0As_34" aria-expanded="true" data-hveid="CBoQAQ"><span style="font-size: 12pt; color: #2a189a; font-family: 'book antiqua', palatino;">`
I'm going to try and remove theses useless informations, there's no "a" link to another page.
